core-domain/src/main/java/org/uniprot/core/unirule/FtagConditionValue.java

```diff
@@ -0,0 +1,7 @@
+package org.uniprot.core.unirule;
+
+import org.uniprot.core.Value;
+
+public interface FtagConditionValue extends Value {
```

Comment:
Not sure about the domain language, but should it FTag...?

Also, does F stand for something? If so, it might be easier to read and for new people to pick up, if F was written in full.